example/src/Examples/Animation/AnimationWithTouchHander2.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { StyleSheet, Dimensions } from "react-native";
+import type { AnimationState, Value } from "@shopify/react-native-skia";
+import {
+  ValueApi,
+  Canvas,
+  Circle,
+  Fill,
+  useTouchHandler,
+  useValue,
+} from "@shopify/react-native-skia";
+
+import { AnimationDemo, Size, Padding } from "./Components";
+
+const { width } = Dimensions.get("window");
+
+const leftBoundary = Size;
+const rightBoundary = width - Size - Padding;
+
+interface PhysicsAnimationState extends AnimationState {
+  velocity: number;
+}
+
+const runBouncing = (translate: Value<number>, initialVelocity: number) => {
```

Comment:
This example was fun to implement. In this example and the one using useValueEffect, it seems that we can somehow make the ball to accelerate if you bounce it fast. I'm sure it's just a math issue nothing else.